src/common/BaseAnnotator.ts

```diff
@@ -101,4 +98,19 @@ export default class BaseAnnotator extends EventEmitter {
     setVisibility = (visibility: boolean): void => {
         this.store.dispatch(setVisibilityAction(visibility));
     };
+
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Just copied the EventEmitter method typings but open to coercing it to fit our usage -- thoughts?